src/main/scala/com/crealytics/spark/excel/ExcelRelation.scala

```diff
@@ -11,6 +11,7 @@ import org.apache.spark.rdd.RDD
 import org.apache.spark.sql._
 import org.apache.spark.sql.sources._
 import org.apache.spark.sql.types._
+import org.apache.hadoop.fs.{FileSystem, Path}
```

Comment:
`FileSystem` is in hadoop-common

```
$ jar -tvf hadoop-common-2.6.0.jar | grep 'org/apache/hadoop/fs/FileSystem.class'
 48255 Thu Nov 13 13:09:30 EST 2014 org/apache/hadoop/fs/FileSystem.class
```

as is `Path`

```
$ jar -tvf hadoop-common-2.6.0.jar | grep 'org/apache/hadoop/fs/Path.class'
 10611 Thu Nov 13 13:09:22 EST 2014 org/apache/hadoop/fs/Path.class
```

hadoop-client has a dependency on hadoop-common.

https://hadoop.apache.org/docs/r2.6.0/hadoop-project-dist/hadoop-client/dependency-analysis.html